test/Quake/combine.qke

```diff
@@ -178,10 +178,14 @@ func.func @d(%c2: i64, %c3: i64, %c1: i64, %off: i16) {
 // CHECK-LABEL:   func.func @d(
 // CHECK-SAME:      %[[VAL_0:.*]]: i64, %[[VAL_1:.*]]: i64, %[[VAL_2:.*]]: i64, %[[VAL_3:.*]]: i16) {
 // CHECK:           %[[VAL_4:.*]] = quake.alloca !quake.veq<4>
-// CHECK:           %[[VAL_5:.*]] = arith.addi %[[VAL_0]], %[[VAL_2]] : i64
+// CHECK:           %[[VAL_0_casted:.*]] = cc.cast unsigned %[[VAL_0]] : (i64) -> i64
+// CHECK:           %[[VAL_2_casted:.*]] = cc.cast unsigned %[[VAL_2]] : (i64) -> i64
```

Comment:
I don't understand why your adding these NOPs.

If we have a 64 bit value of 0xDeadBeefCafeBabe and "sign extend" or "zero extend" it to a 64 bit value we will always get the exact same value, 0xDeadBeefCafeBabe. These casts should be removed as dead code.